src/current/v23.1/connection-pooling.md

```diff
@@ -96,7 +144,7 @@ Connection conn = ds.getConnection();
 
 <section class="filter-content" markdown="1" data-scope="go">
 
-In this example, a Go application similar to the [basic pgx example](build-a-go-app-with-cockroachdb.html) uses the [pgxpool library](https://pkg.go.dev/github.com/jackc/pgx/v4/pgxpool) to create a connection pool on a CockroachDB cluster. The database is being run on 10 cores across the cluster.
+In this example, a Go application similar to the [basic pgx example](build-a-go-app-with-cockroachdb.html) uses the [pgxpool library](https://pkg.go.dev/github.com/jackc/pgx/v4/pgxpool) to create a connection pool on a {{ site.data.products.dedicated }} CockroachDB cluster. The database is being run on 10 cores across the cluster.
```

Comment:
We should update this to `jackc/pgx/v5` and also note that applications using `jackc/pgx` can also include the following query parameters:

* `pool_max_conn_lifetime=300s`
* `pool_max_conn_lifetime_jitter=30s`

https://github.com/jackc/pgx/blob/389931396eea77f4e4b369f659a33b2bff179ae5/pgxpool/pool.go#L270-L286

For example:

```
postgres://my_app@crdb.example.com:26257/crdb_app?sslmode=verify-ca&pool_max_conns=10&pool_max_conn_lifetime=300s&pool_max_conn_lifetime_jitter=30s
```